### PR TITLE
Remove spaces from translatable strings

### PR DESCRIPTION
**Pull request by arnaudbroes**
#825.

@wpsmort you'll just want to have a look at the strings this PR addresses and see if they still look good.

### CODE REVIEW

**Comment by wpsmort** [comment]
This is tested and is good.

File: admin/display/general-metaboxes.php [hunk 1]
```diff
@@ -43,7 +43,7 @@ static function display_extra_metaboxes( $add, $meta ) {
 							<strong>
 								<?php
 								echo aiosp_common::get_upgrade_hyperlink( 'side', __( 'Pro Version', 'all-in-one-seo-pack' ), __( 'CLICK HERE', 'all-in-one-seo-pack' ), '_blank' );
-								echo __( ' to upgrade to Pro Version and get:', 'all-in-one-seo-pack' );
+								printf( ' %1$s', __( 'to upgrade to Pro Version and get:', 'all-in-one-seo-pack' ) );
```
Comment (michaeltorbert) [inline admin/display/general-metaboxes.php]: When there's only one placeholder, there's no need to use %1$s, just %s is fine.

File: modules/aioseop_sitemap.php [hunk 2]
```diff
@@ -649,8 +649,8 @@ public function add_post_types() {
 			$this->default_options['taxonomies']['default']              = array_keys( $this->default_options['taxonomies']['initial_options'] );
 			$this->default_options['excl_categories']['initial_options'] = $this->get_category_titles();
 
-			$post_name = __( ' Post Type', 'all-in-one-seo-pack' );
-			$tax_name  = __( ' Taxonomy', 'all-in-one-seo-pack' );
+			$post_name = sprintf( ' %1$s', __( 'Post Type', 'all-in-one-seo-pack' ) );
```
Comment (michaeltorbert) [inline modules/aioseop_sitemap.php]: These are overly complicated and difficult to read. Something like the following should work just as well.

`$post_name = ' ' . __( 'Post Type', 'all-in-one-seo-pack' ) );`

**Comment by arnaudbroes** [comment]
Comments have been addressed so this can be tested again.

**Comment by wpsmort** [comment]
This is tested and is good.

**Pull request merged**